Source/Core/Core/PowerPC/Gekko.h

```diff
@@ -342,11 +342,11 @@ union UGQR
   explicit UGQR(u32 hex_) : Hex{hex_} {}
 };
 
-#define XER_CA_SHIFT 29
-#define XER_OV_SHIFT 30
-#define XER_SO_SHIFT 31
-#define XER_OV_MASK 1
-#define XER_SO_MASK 2
+#define XER_CA_SHIFT 29u
+#define XER_OV_SHIFT 30u
+#define XER_SO_SHIFT 31u
+#define XER_OV_MASK 1u
+#define XER_SO_MASK 2u
```

Comment:
Would those be candidates for `constexpr`?

Reply:
I guess? Worth a try.